packages/lib-react-components/src/ZooHeader/ZooHeaderContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ZooHeader from './ZooHeader'
+
+export default class ZooHeaderContainer extends React.Component {
+  constructor() {
+    super()
+
+    this.state = {
+      isNarrow: false
+    }
+
+    this.resizeTimeout = NaN
+    this.handleResize = this.handleResize.bind(this)
+  }
+
+  componentDidMount() {
+    this.handleResize()
+    window.addEventListener('resize', this.handleResize)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize)
+  }
+
+  handleResize() {
```

Comment:
I've used [`react-resize-detector`](https://www.npmjs.com/package/react-resize-detector) in the classifier for resize events, which uses https://wicg.github.io/ResizeObserver/ under the hood